components/toggle.tsx

```diff
@@ -2,33 +2,36 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 
-export default function Toggle({onToggle, toggled, onText, offText, disabled}) {
+type Props = {
+    onToggle: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+    toggled: boolean,
+    disabled?: boolean,
+    onText: React.ReactNode,
+    offText: React.ReactNode,
+}
+
+export default function Toggle(props: Props) {
```

Comment:
You can be more explicit here about the type of the function using the `React.FC` type, see an example here: https://github.com/mattermost/mattermost-webapp/blob/master/components/widgets/badges/guest_badge.tsx#L14